datasets/streaming/error_aggregates/reference.md

```diff
@@ -0,0 +1,93 @@
+# Error Aggregates Reference
+
+<!-- toc -->
+
+# Introduction
+
+{% include "./intro.md" %}
+
+# Data Reference
+
+## Sampling
+
+### Invalid Pings
+
+We ignore invalid pings in our processing. Invalid pings are defined as those that:
+
+* The submission dates or activity dates are invalid or missing.
+* The build ID is malformed.
+* The docType field is missing or unknown.
+* The ping does not include a positive count of usage hours.
+
+## Scheduling
+
+The `error_aggregates` job is run continously, using the spark streaming infrastructure
+
+## Schema
+
+The `error_aggregates` table has the following columns which define its dimensions:
+
+* `window_start`: Beginning of time when this sample was taken
+* `window_end`: End of time when this sample was taken (will always be 5 minutes more
+  than `window_start` for any given row)
+* `channel`: the channel, like `release` or `beta`
+* `version`: the version e.g. `57.0.1`
+* `display_version`: like version, but includes beta number if applicable e.g. `57.0.1b4`
+* `build_id`: the YYYYMMDDhhmmss timestamp the program was built, like `20160123180541`. This is also known as the "build ID" or "buildid"
+* `application`: application name (e.g. `Firefox`)
+* `os_name`: name of the OS (e.g. `Darwin` or `Windows_NT`)
+* `os_version`: version of the OS
+* `country`: country code for the user (determined using geoIP), like `US` or `UK`
+* `experiment_id`: identifier of the experiment being participated in, such as `e10s-beta46-noapz@experiments.mozilla.org`, or null if no experiment
```

Comment:
Should we comment that this should always be null since bug 1348748 introduced multiple-experiment annotations?

Reply:
This was a bug in the code, it will be fixed in v2, which will have both `experiment_id` (for backwards compatibility) and `experiments`.